tests/clients/test_environment.py

```diff
@@ -20,33 +20,302 @@
 from marshmallow import ValidationError
 
 from faculty.clients.environment import (
+    Apt,
+    AptPackage,
+    AptPackageSchema,
+    AptSchema,
+    Conda,
+    CondaSchema,
+    Constraint,
     Environment,
     EnvironmentClient,
+    EnvironmentCreateUpdate,
+    EnvironmentCreateUpdateSchema,
+    EnvironmentCreationResponse,
+    EnvironmentCreationResponseSchema,
     EnvironmentSchema,
+    Pip,
+    PipSchema,
+    Python,
+    PythonPackage,
+    PythonPackageSchema,
+    PythonSchema,
+    PythonSpecification,
+    PythonSpecificationSchema,
+    Script,
+    ScriptSchema,
+    Specification,
+    SpecificationSchema,
+    Version,
+    VersionSchema,
 )
 
+VERSION_BODY = {"constraint": "==", "identifier": "1.0.0"}
+VERSION = Version(constraint=Constraint.EQUAL, identifier="1.0.0")
+
+VERSION_BODY_LATEST = "latest"
+VERSION_LATEST = "latest"
+
+INVALID_VERSION_BODY = {"constraint": "==", "identifier": "not_semantic"}
```

Comment:
```suggestion
INVALID_VERSION_BODY = {"constraint": "==", "identifier": "invalid-identifier"}
```
as being SemVer or not isn't relevant here.